harness/determined/exec/tensorboard.py

```diff
@@ -88,5 +87,65 @@ def still_alive() -> bool:
     return p.wait()
 
 
+def get_tensorboard_version(version: str) -> Tuple[str, str]:
+    """
+    Gets the version of the tensorboard package currently installed. Used
+    by downstream processes to determine args passed in.
+    :return: version in the form of (major, minor) tuple
+    """
+
+    major, minor, _ = version.split(".")
+
+    return major, minor
+
+
+def get_tensorboard_args(args: List[str]) -> List[str]:
+    """
+    Builds tensorboard startup args from args passed in from tensorboard-entrypoint.sh
+    Args are added and deprecated at the mercy of tensorboard; all of the below are necessary to
+    support versions 1.14, 2.4, and 2.5
+    """
+    task_id = os.environ["DET_TASK_ID"]
+    port = os.environ["TENSORBOARD_PORT"]
+
+    # Version is passed in from tensorboard-entrypoint.sh which determines the version of
+    # tensorboard running within the started container
+    version = args.pop(0)
+
+    # logdir is the second argument passed in from tensorboard_manager.go. If multiple directories
+    # are specified and the tensorboard version is > 1, use legacy logdir_spec behavior. NOTE:
+    # legacy logdir_spec behavior is not supported by many tensorboard plugins
+    logdir = args.pop(0)
+
+    tensorboard_args = ["tensorboard", f"--port={port}", f"--path_prefix=/proxy/{task_id}", *args]
+
+    major, minor = get_tensorboard_version(version)
+
+    if major == "2":
+        """
+        Tensorboard 2+ no longer exposes all ports. Must pass in "--bind_all" to expose localhost
+        :return: list of startup args passed to tensorboard
+        """
+        tensorboard_args.append("--bind_all")
+        if minor == "5":
+            """
+            Tensorboard 2.5.0 introduces a new experimental feature, fast data loading, which
+            is enabled (load_fast=true) by default. This feature is designed to speed up crawling
+            of logdir files, but prevents plugins from loading correctly. It is disabled here for
+            the Tensorflow profiling plugin (tensorboard-plugin-profile) to work.
+            """
+            tensorboard_args.append("--load_fast=false")
+        if len(logdir.split(",")) > 1:
+            """
+            Tensorboard 2+ no longer accepts multiple comma-delimited directories as logdir.
```

Comment:
This explanation seems redundant to the logdir comment you made above. Can you consolidate them?